.travis.yml

```diff
@@ -4,56 +4,30 @@ sudo: false
 python:
   - "2.7"
   - "3.4"
+  - "3.5"
 
-# command to install dependencies, e.g. pip install -r requirements.txt --use-mirrors
+cache:
```

Comment:
Regarding conda, all of it is cached. The installation script is removed upon successful install and is not downloaded as long as the cached miniconda exists.